src/common/settings.js

```diff
@@ -508,6 +508,13 @@ const character_settings = {
 
     // effects
 
+    "effects-exhaustion": {
```

Comment:
Should be `effects-exhaustion-2024` as the 2014 rules for exhaustion also affect rolls, so I think we ought to have both 2014 and 2024 exhaustion effects supported.
Unless... we check if the character sheet is 2024 or 2014 and apply the exhaustion rules depending on the character sheet version, in which case the description needs to be version agnostic.

Reply:
yeah i kept the exhaustion not checking for version because someone might want to use 2024 exhaustion on a 2014 character